packages/discovery-provider/ddl/migrations/0053_playlists_tracks_relations.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS collection_track_relations (
+  collection_id INTEGER NOT NULL,
+  track_id INTEGER NOT NULL,
+  is_delete BOOLEAN NOT NULL,
```

Comment:
does this indicate track is deleted, collection is deleted or either?

Reply:
indicates track is removed from playlist

Reply:
thanks, i think a comment somewhere would be ideal, cause is_delete doesn't really explain which entity is deleted, and also delete vs removed a bit confusing as well.

Reply:
good call! yeah maybe `is_delete` is overloaded.. for tracks `is_delete` does indicate the track was deleted. maybe `is_remove`.